app/src/main/java/com/stevesoltys/seedvault/settings/ExpertSettingsFragment.kt

```diff
@@ -16,10 +16,10 @@ class ExpertSettingsFragment : PreferenceFragmentCompat() {
     private val viewModel: SettingsViewModel by sharedViewModel()
     private val packageService: PackageService by inject()
 
-    // TODO set mimeType when upgrading androidx lib
-    private val createFileLauncher = registerForActivityResult(CreateDocument()) { uri ->
-        viewModel.onLogcatUriReceived(uri)
-    }
+    private val createFileLauncher =
+        registerForActivityResult(CreateDocument("text/plain")) { uri ->
```

Comment:
`org.apache.http.entity.TEXT_PLAIN.mimeType` maybe or something similar

Reply:
I found `com.google.android.mms.ContentType.TEXT_PLAIN` and now use that, but not sure if that's better than just leaving the string in.